website/docs/r/kusto_cluster.html.markdown

```diff
@@ -68,7 +68,9 @@ The following arguments are supported:
 
 * `purge_enabled` - (Optional) Specifies if the purge operations are enabled.
 
-* `virtual_network_configuration` - (Optional) A `virtual_network_configuration` block as defined below. Changing this forces a new resource to be created.
+* `virtual_network_configuration` - (Optional) A `virtual_network_configuration` block as defined below.
+
+~> **NOTE:** Currently removing `virtual_network_configuration` sets the `virtual_network_configuration` to `Disabled` state. But any changes to `virtual_network_configuration` in `Disabled` state forces a new resource to be created.
```

Comment:
This seems togo against how terraform is intended to work. i think we should just do a 2nd update to set what is in config if the api changes it?

Reply:
The simplest way to solve this is adding a `state` property to this block like https://github.com/hashicorp/terraform-provider-azurerm/pull/23912?

Reply:
sorry I left this comment in the wrong place about the wrong thing